api/src/navLinks/navLinks.controller.ts

```diff
@@ -0,0 +1,24 @@
+import { Controller, Get, Post, Body, Param } from '@nestjs/common';
+import { CreateLinksDto } from './dto/create-navLinks.dto';
+import { LinksService } from './navLinks.service';
+import { Links } from '../types/navLinks';
+
+@Controller('links')
+export class LinksController {
+  constructor(private readonly linksService: LinksService) {};
+
+  @Get()
+  findAll(): Promise<Links[]> {
+    return this.linksService.findAll();
+  }
+
+  @Get(':id')
+  async findOne(@Param('id') id): Promise<Links> {
+    return this.linksService.findOne(id);
+  }
+
+  @Post()
```

Comment:
remove write apis. 

we should add the links manually for now, once we implemented the auth, we can start the write apis

Reply:
ok that was discussed was Abdullah after submitting this pr 
changes will be done.